src/cpp/session/resources/pagedtable/pagedtable.js

```diff
@@ -525,6 +528,36 @@ var PagedTable = function (pagedTable) {
 
     var fragment = document.createDocumentFragment();
 
+    // show metadata on top of header
+    if (options.metadata !== {}) {
```

Comment:
This check feels a bit awkward, since I don't think there's any real guarantee that two empty objects are identical:

```
$ node
Welcome to Node.js v13.13.0.
Type ".help" for more information.
> {} === {}
false
```

Maybe instead check for the length of `Object.keys(options.metadata)`?

Reply:
I did exactly this last week in 76e04d93d1a0799305392c4502384ebf016f86c3 -- no idea why GH showed you the old diff!